ansible/group_vars/all

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
You might try renaming all to `all.yml` so you don't have to do any editor hijinks to make it show the correct syntax highlighting.